edgelet/contrib/enterprise-linux/00-aziot-edged.preset

```diff
@@ -0,0 +1,4 @@
+# Enable aziot-edged to start automatically. The service should start if data is written to socket.
+
+enable aziot-edged.mgmt.socket
```

Comment:
I see that for centos, we enable aziot-edged.service , Why do we have a different approach for RHEL?

Reply:
This should be sufficient, but it couldn't hurt to add it to be safe.